assets/src/components/child/Child.js

```diff
@@ -0,0 +1,45 @@
+import zdClient from "./../../libs/ZDClient.js";
+import childTemplate from "./ChildTemplate.js";
+import { splitIdentities } from "./../../libs/Helper.js";
+
+const Child = {
+  template: childTemplate,
+  props: {
+    dictionary:Object,
+    ticket:Object,
+    zd_instance:String
+  },
+  components: {},
+  data() {
+    return {};
+  },
+  created() {
+    const { dictionary, ticket } = this;
+    if (ticket.has_requester) {
+      let identities = ticket.data.requester.identities;
+      let splittedIdentities = splitIdentities(identities);
+      // TODO: Auto search by identities
+    } else {
+      zdClient.invoke.notify(dictionary.messages.no_ticket_requester);
+    }
+  },
+  mounted() {},
+  updated() {},
+  computed: {},
+  methods: {
+    openModal() {
+      const { client } = zdClient.app;
```

Comment:
sorry I'm a bit against adding the modal il the boilerplate. we could instead create a script where we ask for app requirements and if the app requires a modal, adding it via script. what do you think?